xbps-src

```diff
@@ -473,10 +473,22 @@ readonly REQHOST_UTILS_BOOTSTRAP="make gawk bash bison sed gcc g++ msgfmt makein
 
 # Required utilities in host system for chroot ops.
 readonly REQHOST_UTILS="xbps-install xbps-query xbps-rindex xbps-uhelper \
-    xbps-reconfigure xbps-remove xbps-create xbps-uchroot xbps-uunshare git"
+    xbps-reconfigure xbps-remove xbps-create xbps-uchroot xbps-uunshare"
 
 check_reqhost_utils
 
+# Try using chroot-git then git from the host system
+if command -v chroot-git &>/dev/null; then
+	XBPS_GIT_CMD=$(command -v chroot-git)
+elif command -v git &>/dev/null; then
+	XBPS_GIT_CMD=$(command -v git)
```

Comment:
Why not just =git?

Reply:
Can be, was just following what is already done on common/hooks/pre-install/05-generate-gitrevs.sh and common/environment/setup/git.sh